#goal
float32 fwd_velocity               # The speed at which the robot should move forwards (m/s)
float32 stopping_distance          # Minimum distance of an approaching obstacle before the robot stops (meters) 
---
#result
float32 total_distance_travelled   # Total distance travelled during the action (meters)
float32 closest_obstacle           # LaserScan distance to the closest detected obstacle up ahead (meters)
---
#feedback
float32 current_distance_travelled # Distance travelled so far during the action (meters)